meta-dts-distro/recipes-bsp/txe-secure-boot/smmstoretool_git.bb

```diff
@@ -0,0 +1,25 @@
+DESCRIPTION = "Offline SMMSTORE variable modification too"
+LICENSE = "GPL-2.0-or-later"
+
+SRC_URI = " \
+git://github.com/coreboot/coreboot.git;branch=main;protocol=https \
+file://a5df001.diff;patchdir=${WORKDIR}/git \
+"
+
+SRCREV = "602653abed391ae1b1445ad86d0f05b8b5b678cb"
+
+LIC_FILES_CHKSUM = " \
+file://${WORKDIR}/git/LICENSES/GPL-2.0-or-later.txt;md5=261bea1168c0bdfa73232ee90df11eb6 \
+"
+
+inherit pkgconfig
+
+S = "${WORKDIR}/git/util/smmstoretool"
+
+do_install() {
+    oe_runmake 'DESTDIR=${D}' install
+}
+
+FILES:${PN} += " \
+   /usr/local/bin/smmstoretool \
```

Comment:
Why not ${bindir} ?

Reply:
`${bindir}` expands to `/usr/bin`, but the [Makefile ](https://github.com/coreboot/coreboot/blob/main/util/smmstoretool/Makefile#L11) installs the tool in `/usr/local/bin` by default. Is that location okay or should I change it?

Reply:
You should ovveride the PREFIX variable in Yocto recipe to adhere to typical paths in Yocto.

Reply:
Is this what you mean? 33cda5107591077ab6db0d141c5a911f29130b2d

Reply:
Definitely not. Please take a look at other recipes (e.g. cbmem), even in meta-dts. It is very common pattern.

Reply:
Done f26221db11a05bcbede4bc06596a5983acb02373